src/algoliasearch.helper.js

```diff
@@ -587,18 +587,18 @@ AlgoliaSearchHelper.prototype.toggleExclude = function() {
  * @fires change
  * @chainable
  */
-AlgoliaSearchHelper.prototype.toggleRefinement = function(facet, value) {
```

Comment:
Can you add an alias method from toggleRefinement to toggleFacetRefinement? So that implementations using toggleRefinement don't break.